.github/workflows/helm-enterprise.yml

```diff
@@ -117,8 +117,7 @@ jobs:
         run: |
           if [ "${{ matrix.values }}" = "values.yaml" -o "${{ matrix.values }}" = "pre-release_values.yaml" ]; then
             echo "Enabling clustered tests via auto-scaling: 2 replicas max, 10s stabilization window"
-            echo "helm_install_params=--values test/autoscaling-hpa-test-values.yaml \
-              --set dtas.additionalArgs[0]='-k not test_renditions'" >> "$GITHUB_OUTPUT"
```

Comment:
We removing this to make sure its overloaded?

Reply:
I think it was a mistake to have this on master, maybe was a leftover from tests

the renditions skip was really necessary for acs 7.0 - and we dropped support for it in the meanwhile